core/src/main/java/org/frankframework/pipes/ForEachChildElementPipe.java

```diff
@@ -72,36 +65,29 @@
  * @since 4.6.1
  */
 @Category(Category.Type.BASIC)
-public class ForEachChildElementPipe extends StringIteratorPipe implements IThreadCreator {
+public class ForEachChildElementPipe extends StringIteratorPipe {
 
 	public static final int DEFAULT_XSLT_VERSION = 1; // currently only Xalan supports XSLT Streaming
 
-	private @Getter boolean processFile=false;
+	private @Getter boolean processFile = false;
 	private @Getter String containerElement;
 	private @Getter String targetElement;
-	private @Getter String elementXPathExpression=null;
-	private @Getter int xsltVersion=DEFAULT_XSLT_VERSION;
-	private @Getter boolean removeNamespaces=true;
-	private boolean streamingXslt;
-
-	private TransformerPool extractElementsTp=null;
-	private @Setter ThreadLifeCycleEventListener<Object> threadLifeCycleEventListener;
-	private @Setter IThreadConnectableTransactionManager<?,?> txManager;
+	private @Getter String elementXPathExpression = null;
+	private @Getter int xsltVersion = DEFAULT_XSLT_VERSION;
+	private @Getter boolean removeNamespaces = true;
+
+	private TransformerPool extractElementsTp = null;
 	private @Getter @Setter IXmlDebugger xmlDebugger;
 
 	@Override
 	public void configure() throws ConfigurationException {
 		super.configure();
 		try {
 			if (StringUtils.isNotEmpty(getElementXPathExpression())) {
-				streamingXslt = AppConstants.getInstance(getConfigurationClassLoader()).getBoolean(XmlUtils.XSLT_STREAMING_BY_DEFAULT_KEY, false);
-				if (getXsltVersion()==0) {
+				if (getXsltVersion() == 0) {
 					setXsltVersion(DEFAULT_XSLT_VERSION);
 				}
-				if (streamingXslt && getXsltVersion() != DEFAULT_XSLT_VERSION) {
-					ConfigurationWarnings.add(this, log, "XsltProcessor xsltVersion ["+getXsltVersion()+"] currently does not support streaming XSLT, might lead to memory problems for large messages", SuppressKeys.XSLT_STREAMING_SUPRESS_KEY);
```

Comment:
Ik denk dat deze melding nog wel handig is, alleen Xslt 1 kan streaming dingen verwerken dacht ik. Saxon kan het ook wel maar dan kost het geld...